app/components/TrackCard/index.js

```diff
@@ -53,13 +64,18 @@ const CustomButton = styled.div`
 `;
 
 export function TrackCard({ item, currentPlayingId, isPlaying, onPlay, onPause }) {
-  const { artworkUrl100, trackName, artistName, previewUrl, trackId } = item;
+  const { trackName, artistName, artworkUrl100, previewUrl, trackId } = item;
   return (
-    <CustomCard data-testid="track-card" cover={<Image alt="artwork" src={artworkUrl100} />}>
-      <TextCard data-testid="text-card" title={trackName} description={artistName} />
+    <CustomCard data-testid="track-card">
+      <Link to={`/track/${trackId}`}>
+        <ImageCard>
+          <Image alt="artwork" src={artworkUrl100} />
+          <TextCard data-testid="text-card" title={trackName} description={artistName} />
+        </ImageCard>
+      </Link>
       <CustomButton>
```

Comment:
What is the <CustomRight/> component below?

Reply:
for play pause buttons